lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1046,7 +1052,9 @@ def _initialize_credential(self) -> None:
 
     def _load_template(self) -> Any:
         if self._arm_template is None:
-            template_file_path = Path(__file__).parent / "arm_template.json"
+            template_file_path = (
```

Comment:
How about existing code, which instruments in the template? Are they effected? Including the internal nsg hook?

Reply:
I have tested it, nsg hook works well in both scenario.